modules/rest-api/pages/rest-cluster-autofailover-enable.adoc

```diff
@@ -47,6 +47,11 @@ Specifies the number of seconds that must elapse, with a node unavailable, befor
 Default setting is 120.
 The `timeout` parameter can only be specified when `enabled` is set to `true`.
 This parameter and its values are ignored if the value for the `enabled` parameter is `false`.
++
+The value of `timeout` can also be specified as 1 second.
```

Comment:
I don't know if we would want to call out the difference in behaviour in the health monitoring between 1-5s timeout values and 5s-xyzs timeout values. Whilst we would see higher CPU usage between 1s and 5s (the lower we set the value, the higher the CPU usage will be), we should see no difference in CPU usage between 5s and 120s for example.

Reply:
Okay, I may not be reading the text in the same way, but I don't have the sense that we are making that comparison. Anyway, I've change the explanation,so that it's quite different now. So take a look at my revision and let me know what you think.